src/SASjs.ts

```diff
@@ -107,6 +107,34 @@ export default class SASjs {
     return await this.sasViyaApiClient!.getExecutableContexts(accessToken)
   }
 
+  public async createContext(
```

Comment:
all public functions need to be documented

Reply:
I've added comments for the new functions I added - but if you look closely, none of the other public functions in this file are documented either.
We'll need to address that as part of a separate issue.